gradle.properties

```diff
@@ -28,3 +28,6 @@ android.suppressUnsupportedCompileSdk=UpsideDownCakePrivacySandbox
 
 # Signals to our own plugin that we are building within the repo.
 app.cash.paparazzi.internal=true
+
+# Gradle
+org.gradle.configuration-cache.parallel=true
```

Comment:
doesn't apply to CI clean builds

Reply:
But it would do for any contributor cloning the repository? There are a lot of modules, because of test projects. Wouldn't it be beneficial to share the win with everyone and configure the CI to be the exception?

(I'm curious in general, about the approach you take wrt sharing config that helps developers)

Reply:
Yep my thought was it would be an improvement for contributors. I think your suggestion of adding a gradle flag to not use config cache makes sense if we enable this.